lib/engine/round/base.rb

```diff
@@ -294,6 +294,10 @@ def lay_tile(action)
         end
       end
 
+      def game_error_not_upgradeable(old_tile, tile)
```

Comment:
this change is unnecessary, if you did this due to linters, just do multiline unless statement

Reply:
Fixed in later versions.